test/integration/utils.go

```diff
@@ -0,0 +1,21 @@
+package integration
+
+import (
+	"math/rand"
+	"time"
+)
+
+var seededRand = rand.New(rand.NewSource(time.Now().UnixNano()))
+
+func stringWithCharset(length int, charset string) string {
+	b := make([]byte, length)
+	for i := range b {
+		b[i] = charset[seededRand.Intn(len(charset))]
+	}
+	return string(b)
+}
+
+func randomNamespace() string {
+	charset := "abcdefghijklmnopqrstuvwxyz"
```

Comment:
We can do random namespaces, but I know it is a frustration sometimes in tests to not know what test created the ns.

Would it be better to just use `"sonobuoy-"+t.Name()` as a namespace if we needed it?